ITISFoundation/osparc-simcore
# [bugfix] list of studies disappear (tags)

## What do these changes do?


Solves bug where the list of studies was disappearing when modifying the tags **and** adds regression tests using puppeteer.

## Related issue number


closes https://github.com/ITISFoundation/osparc-issues/issues/379

## How to test


Check https://github.com/ITISFoundation/osparc-issues/issues/379 no longer happens and run e2e tests with ``npm test`` inside ``/tests/e2e``.

## Checklist

- [x] Tests for the changes exist
